Framework/include/QualityControl/CheckRunner.h

```diff
@@ -114,7 +114,7 @@ class CheckRunner : public framework::Task
   void setTaskStoreSet(std::unordered_set<std::string> storeSet) { mInputStoreSet = storeSet; }
   std::string getDeviceName() { return mDeviceName; };
 
-  static framework::DataProcessorLabel getLabel() { return { "qc-check-runner" }; }
+  static framework::DataProcessorLabel getCheckLabel() { return { "qc-check-runner" }; }
```

Comment:
Hmm, it is not a Check label, but CheckRunner's, isn' it?

Reply:
yep